tuning/sft_trainer.py

```diff
@@ -360,7 +361,24 @@ def train(
         for x in framework.get_callbacks_and_ready_for_train(model, accelerator):
             trainer.add_callback(x)
 
-    trainer.train()
+    resume_from_checkpoint = None
+    # Check if resume flag is not passed (None), or if flag is true and
+    # output_dir has checkpoints then get last checkpoint from output_dir
+    if (
+        training_args.resume_from_checkpoint is None
```

Comment:
Is this saying that if resume_from_checkpoint is not set, then it would enable resume from checkpoint? That seems off...If a user doesn't pass anything in, why would it be enabled?

Reply:
Yes, I can update the Readme. In this [readme](https://github.com/Abhishek-TAMU/fms-hf-tuning/blob/main/README.md), is there any specific section recommended, where I can mention above details for users (regarding info of `resume_from_checkpoint` flag) ?